.github/workflows/codestyle.yml

```diff
@@ -0,0 +1,32 @@
+name: Code quality
+
+on: [push, pull_request]
+
+jobs:
+  eslint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - run: |
+          npm install eslint --save-dev
+          curl -O http://diracproject.web.cern.ch/diracproject/externalLibraries/@sencha_eslint-plugin-extjs.7.0.0.tgz
+          npm install @sencha_eslint-plugin-extjs.7.0.0.tgz --save-dev
+          node_modules/.bin/eslint WebApp
+  pyLint:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Prepare environment
+      run: |
+        conda env create --name webapp-testing environment.yml
```

Comment:
Last change: put a python version matrix at the beginning of this file (with, right now, only python 2.7.13), and use such version in this line (and then remove the python version from the environment.yml)

Reply:
done